packages/components/src/components/hds/tooltip-button/index.ts

```diff
@@ -48,10 +52,10 @@ export default class HdsTooltipIndexComponent extends Component {
 
     return {
       ...this.args.extraTippyOptions,
-      // takes string
       placement: placement,
-      // takes array of 2 numbers (skidding, distance): array(0, 0)
-      offset: this.args.offset,
+      showOnCreate: true,
+      // takes array of 2 numbers (skidding, distance): array(0, 10)
+      offset: this.args.offset ? this.args.offset : [0, 10],
```

Comment:
Note: turns out the default if offset is not provided is [0,10] not [0,0]

https://github.com/atomiks/tippyjs/blob/ad85f6feb79cf6c5853c43bf1b2a50c4fa98e7a1/src/props.ts#L44

Reply:
Nice catch! 👏